pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -728,6 +728,8 @@ private IDockableForm DeserializeForm(string persistentString)
                     return _listGraphPeakArea.FirstOrDefault(g => g.Type == type) ?? CreateGraphPeakArea(type);
                 else if (split[1] == typeof(MassErrorGraphController).Name)
                     return _listGraphMassError.FirstOrDefault(g => g.Type == type) ?? CreateGraphMassError(type);
+                else if (split[1] == typeof(DetectionsGraphController).Name)

```

Comment:
Better if you give split[0], split[1], and split[2] descriptive names than using them this much through so many lines of code.